features/export.feature

```diff
@@ -16,3 +16,51 @@ Feature: cli-kintone export command
     And The output message should match with the pattern: "\"\d+\",\"Alice\",\"10\""
     And The output message should match with the pattern: "\"\d+\",\"Bob\",\"20\""
     And The output message should match with the pattern: "\"\d+\",\"Jenny\",\"30\""
+
+  Scenario: CliKintoneTest-78 Should return the error message when the user has no privilege to view records
+    Given Load app ID of the app "app_for_export" as env var: "APP_ID"
+    And Load username and password of the app "app_for_export" with exact permissions "add" as env vars: "USERNAME" and "PASSWORD"
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --username $USERNAME --password $PASSWORD"
+    Then I should get the exit code is non-zero
+    And The output error message should match with the pattern: "ERROR: \[403\] \[CB_NO02\] No privilege to proceed."
+
+  Scenario: CliKintoneTest-79 Should return the record contents of the app in the space in CSV format
+    Given The app "app_in_space" has no records
+    And The app "app_in_space" has some records as below:
+      | Text   | Number |
+      | Alice  | 10     |
+      | Bob    | 20     |
+      | Jenny  | 30     |
+    And Load app ID of the app "app_in_space" as env var: "APP_ID"
+    And Load app token of the app "app_in_space" with exact permissions "view" as env var: "API_TOKEN"
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN"
+    Then I should get the exit code is zero
+    And The app "app_in_space" should has records as below:
```

Comment:
@tuanphamcybozu 

The assert step should be based on the output of the export command (in step line 36).
Please help me to check again.
Thank you.